app/livechat/server/methods/transfer.js

```diff
@@ -6,23 +6,26 @@ import { Livechat } from '../lib/Livechat';
 
 Meteor.methods({
 	'livechat:transfer'(transferData) {
-		if (!Meteor.userId() || !hasPermission(Meteor.userId(), 'view-l-room')) {
+		if (!transferData.userLoggedIn || !hasPermission(transferData.userLoggedIn, 'view-l-room')) {
```

Comment:
This change is not allowed.
Maybe you'll need to change your logic, like sending back to the original agent the request response and then they will call the `livechat:transfer` method by from their side.